Directory.Build.props

```diff
@@ -13,14 +13,25 @@
 		<Authors>Paul den Dulk, Felix Obermaier</Authors>
 		<Company>BruTile Developers Team</Company>
 		<LangVersion>10.0</LangVersion>
-		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+		<TreatWarningsAsErrors>false</TreatWarningsAsErrors>

```

Comment:
I would like to keep TreatWarningsAsErrors true. It saves a lot of trouble in the long run. Not sure what you are trying to suppress. Could a global suppression be added that applies to everything regarding this one specific rule?

Reply:
This wasn‘t intentional I had to disable treat warnings as errors so that I could suppress the warning. Will restore the warnings as errors.